src/main.rs

```diff
@@ -23,6 +23,10 @@ struct Cli {
 
 #[derive(Debug, Subcommand)]
 enum Commands {
+    #[command(
+        about = "displays info about the farmer instance (i.e. total amount of rewards, and status of initial plotting)"
+    )]
+    Info,
```

Comment:
A side note. I think we should let users to supply their own path to the config file

Reply:
Yeah

Reply:
do you mean the config file or the summary file? If it is for summary, it's a temporary solution and will be evicted in the future as we discussed before. 
If for the config, I don't think we should let users to choose their own config path. Because then we would need another persistent storage to store this info, to be able to read the config file. And best place for that persistent storage will probably be the default config location